traffic_ops/testing/api/v3/cachegroups_test.go

```diff
@@ -42,11 +42,43 @@ func TestCacheGroups(t *testing.T) {
 		var header http.Header
 		header = make(map[string][]string)
 		header.Set(rfc.IfModifiedSince, time)
+		header.Set(rfc.IfUnmodifiedSince, time)
```

Comment:
So currently, we support `If-Modified-Since` in only the `GET` requests, whereas `If-Unmodified-Since` and `If-Match` are supported only in the `PUT` requests. I have set them to the same time (a time in the past) for both the headers, because these headers are being used in different tests, and have no relation to each other as such.

Reply:
I see.